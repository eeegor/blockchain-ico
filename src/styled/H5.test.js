import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import { H5 } from '.';

describe('H5', () => {
	test('renders default h1', () => {
		const tree = renderer.create(<H5>Title</H5>).toJSON();
		expect(tree).toMatchSnapshot();
	});

	test('renders h1 with custom color', () => {
		const tree = renderer.create(<H5 color="green">Title</H5>).toJSON();
		expect(tree).toMatchSnapshot();
		expect(tree).toHaveStyleRule('color', 'green');
	});
});
