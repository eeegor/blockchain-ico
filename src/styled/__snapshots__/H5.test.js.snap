// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`H5 renders default h1 1`] = `
.c0 {
  font-size: 17px;
  line-height: 25px;
  font-family: GTCinetype-Light;
  margin-bottom: 12px;
}

<h5
  className="c0"
>
  Title
</h5>
`;

exports[`H5 renders h1 with custom color 1`] = `
.c0 {
  font-size: 17px;
  line-height: 25px;
  font-family: GTCinetype-Light;
  margin-bottom: 12px;
  color: green;
}

<h5
  className="c0"
  color="green"
>
  Title
</h5>
`;
